governance/polls/Initial Parameters for StarkNet Simple Bridge - March 28, 2022.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Initial Parameters for StarkNet Simple Bridge - March 28, 2022
+summary: Signal your support or opposition to the following initial parameters, designed for the deployment of StarkNet Simple Bridge.
```

Comment:
Bit nitpicky, but 'designed' maybe isn't the right word here. "Signal your support or opposition to the initial parameters that will be used in the deployment of StarkNet Simple Bridge."